src/structures/ThreadChannel.js

```diff
@@ -277,9 +277,9 @@ class ThreadChannel extends Channel {
   async edit(data, reason) {
     if (data.autoArchiveDuration === 'MAX') {
       data.autoArchiveDuration = 1440;
-      if (this.guild.premiumSubscriptionCount >= 15) {
+      if (this.channel.guild.features.includes('SEVEN_DAY_THREAD_ARCHIVE')) {
         data.autoArchiveDuration = 10080;
-      } else if (this.guild.premiumSubscriptionCount >= 2) {
+      } else if (this.channel.guild.features.includes('THREE_DAY_THREAD_ARCHIVE')) {
```

Comment:
Ditto

Reply:
haha yeah, fixing rn, missed it in copypaste